### ISSUE TITLE
About breaking change in CLI `tetra version` from `--client` to `--server`

### ISSUE THREAD

**Issue by lucasfcnunes** [cilium/tetragon#1418]
### What happened?

This commit introduces a change in the tetra CLI that makes it incompatible with cilium (v1, stable) that works with the `cilium version --client` pattern.
This issue might be of use to future cilium v2 release.

*There wasn't a PR so I opened this issue.

from: https://github.com/cilium/tetragon/commit/4de6d5458e9a370d1113924276d6c96810465d37
to: https://github.com/cilium/tetragon/commit/ea1fdd60f52370fb5dfd0b189d64ae4061945ab0

### Tetragon Version

0.11.0

### Kernel Version

NA

**Comment by mtardy** [comment]
Indeed, I changed the behavior so that `tetragon version` is by default `tetragon version --client`. The old behavior is now `tetragon version --server`. I wasn't aware that this was related to `cilium` CLI, what's the incompatibility issue exactly?

**Comment by lucasfcnunes** [comment]
@mtardy
Cilium has too the `cilium version --client` command.

Current behavior (after https://github.com/cilium/tetragon/commit/ea1fdd60f52370fb5dfd0b189d64ae4061945ab0):
```sh
# implicit (odd one and the cause of this issue)
cilium version          # server
tetra version           # client
# explicit
cilium version --client # client
tetra version --server  # server
```

**Comment by kkourt** [comment]
The fact that we ended up with a different behavior that the cilium CLI is unfortunate, but I'm not sure that it is a bug. 

@lucasfcnunes is there a practical reason that for `tetra` to be consistent with the cilium cli? Feel free to submit a PR that addresses this.

**Comment by kkourt** [comment]
> @kkourt I agree it's not a real bug. Guess I should've opened it as a blank issue.

No worries, I just wanted to explain why I removed the label.

> I opened this to try to reverse it on tetra v0 or change it in future cilium releases (v1 -> v2). This broke one of my CI/CD scripts so I thought of bringing it up to debate. The practical reason is having to guess how it all works. I personally only check for client version and I think the current behavior of tetra v0.11.0 is better and intuitive.

Thanks! FWIW, I don't have a strong opinion on the matter. For some context, my motivation for having both the client and the server version (in the old version) was to spot incompatibilities between client/server versions.

We have no guarantees for the stability of the tetra CLI, but we should maybe be more careful about introducing breaking changes in the future if we can avoid it.
